fastly/resource_fastly_service_acl_entries_v1.go

```diff
@@ -74,6 +76,11 @@ func resourceServiceAclEntriesV1() *schema.Resource {
 	}
 }
 
+func convertSubnetToInt(s string) int {
```

Comment:
I think normally helper functions such as these go at the bottom of the file

Reply:
Thanks! Moved that to the bottom